### ISSUE TITLE
Typed holes

### ISSUE THREAD

**Issue by japgolly** [lampepfl/dotty-feature-requests#70]
It would be great to have typed holes in Scala

* https://wiki.haskell.org/GHC/Typed_holes

**Comment by smarter** [comment]
Agreed. I did a quick prototype recently which I showed off at scalasphere (talk not online yet), hopefully I'll find time to finish that.
